pyiron/atomistics/structure/atoms.py

```diff
@@ -331,6 +330,51 @@ def set_high_symmetry_points(self, new_high_symmetry_points):
             raise ValueError("has to be dict!")
         self._high_symmetry_points = new_high_symmetry_points
 
+    def add_high_symmetry_points(self, new_points):
+        """
+        Adds new points to the dict of existing high symmetry points.
+
+        Args:
+            new_points (dict): Points to add
+        """
+        if self.get_high_symmetry_points() is None:
+            raise AssertionError("Construct high symmetry points first. Use self.band_structure_calculations().")
```

Comment:
`self.band_structure_calculation` and not `self.band_structure_calculations`

Reply:
done